scripts/webpack/webpack.base.babel.js

```diff
@@ -15,17 +15,17 @@ process.env.REACT_WEBEX_VERSION = version;
 module.exports = (options, env) => {
   const packageJson = require('../../package.json');
   const plugins = [
-    new webpack.EnvironmentPlugin([
-      'NODE_ENV',
-      'WEBEX_CLIENT_ID',
-      'REACT_WEBEX_VERSION',
-      'WDM_SERVICE_URL',
-      'IDBROKER_BASE_URL',
-      'WEBEX_TEST_USERS_CONVERSATION_SERVICE_URL',
-      'WEBEX_CONVERSATION_DEFAULT_CLUSTER',
-      'FEDERATION',
-      'U2C_SERVICE_URL'
-    ]),
+    new webpack.EnvironmentPlugin({
+      NODE_ENV: "production",
+      REACT_WEBEX_VERSION: "",
+      FEDERATION: "",
+      IDBROKER_BASE_URL: "",
+      U2C_SERVICE_URL: "",
+      WEBEX_CLIENT_ID: "",
+      WDM_SERVICE_URL: "",
+      WEBEX_CONVERSATION_DEFAULT_CLUSTER: "",
+      WEBEX_TEST_USERS_CONVERSATION_SERVICE_URL: "",
+    }),
```

Comment:
After moving to yarn workspaces, there is a different way in which env variables are passed. This change does nothing but provides a fallback to these empty string just in case these variables are not available. Im still trying to figure out how yarn workspaces work with env varibales. 

A possible speculation is that previously all the commands were run from root folder and the .env folder was also present in the root folder. But now when we run yarn build for each workspace the root folder becomes the workspace and there is no .env available there.